release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4001,4 +4001,24 @@ $ oc adm release info 4.12.30 --pullspecs
 [id="ocp-4-12-30-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+
+[id="ocp-4-12-31"]
+=== RHBA-2023:4756 - {product-title} 4.12.31 bug fix update
+
+Issued: 2023-08-31
+
+{product-title} release 4.12.31, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.
```

Comment:
Slight edit for readability cutting down on the number of word.

`{product-title} release 4.12.31 includes security updates. The list of bug fixes included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.`

Is the last sentence necessary? It seems to repeat what's in the previous sentence.

Reply:
@nalhadef this is stock text for the RNs so if this is edited they would have to edit all entries. Also, to answer your question, yes the the last sentence is necessary as it is the link for the RPMs. The other link is for the images. Hope that helps!

Reply:
Oh, sorry. I missed this thread. Thanks for answering, @opayne1 .